jobs/JEVS_NWPS_PLOTS

```diff
@@ -59,6 +59,8 @@ export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}
 export EVSINnwps=${COMIN}/stats/${COMPONENT}/${MODELNAME}.${VDATE}
 export COMOUTplots=${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}
+export COMINnwps=${COMINnwps:-$(compath.py ${envir}/com/${MODELNAME}/${nwps_ver})}
+export COMINhist=${COMINnwps}/${MODELNAME}.${VDATE}/nwges/dcom_hist.txt
```

Comment:
When NCO sees `COMINhist`, they'll think it is the com directory for a model name hist. Where is in the code it uses `COMINhist`, can it use `${COMINnwps}/${MODELNAME}.${VDATE}/nwges/dcom_hist.txt` instead?

Reply:
@malloryprow, I had initially added the path to com_hist.txt but later I decided to use that in a later PR when I add a list of all WFOs. So, I removed the line that defines it for now.